roachtest: weekly/tpcc-max failed

SHA: https://github.com/cockroachdb/cockroach/commits/cf5c2bd2372e633d2f63e08e5bffca7c2a7ec59f

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=weekly/tpcc-max PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1510122&tab=artifacts#/weekly/tpcc-max

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20190927-1510122/weekly/tpcc-max/run_1
	cluster.go:1764,tpcc.go:183,cluster.go:2120,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1569562392-03-n4cpu16:4 -- ./workload run tpcc --warehouses=1350 --histograms=perf/stats.json  --ramp=5m0s --duration=144h0m0s {pgurl:1-3} returned:
		stderr:
		
		stdout:
		  0.0      0.0      0.0 delivery
		26049.0s        0            1.0          269.7 103079.2 103079.2 103079.2 103079.2 newOrder
		26049.0s        0            0.0           27.0      0.0      0.0      0.0      0.0 orderStatus
		26049.0s        0            0.0          269.7      0.0      0.0      0.0      0.0 payment
		26049.0s        0            0.0           27.0      0.0      0.0      0.0      0.0 stockLevel
		26050.0s        0            0.0           27.0      0.0      0.0      0.0      0.0 delivery
		26050.0s        0            0.0          269.7      0.0      0.0      0.0      0.0 newOrder
		26050.0s        0            0.0           27.0      0.0      0.0      0.0      0.0 orderStatus
		26050.0s        0            0.0          269.7      0.0      0.0      0.0      0.0 payment
		26050.0s        0            0.0           27.0      0.0      0.0      0.0      0.0 stockLevel
		Error: error in stockLevel: ERROR: communication error: rpc error: code = Canceled desc = context canceled (SQLSTATE 08006)
		Error:  exit status 1
		: exit status 1

```

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656753&tab=buildLog) on [master@bf8d6db2ef9b66d301996cbb8ebbb15b7c978d1a](https://github.com/cockroachdb/cockroach/commits/bf8d6db2ef9b66d301996cbb8ebbb15b7c978d1a):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191220-1656753/weekly/tpcc-max/run_1
	cluster.go:1786,tpcc.go:157,cluster.go:2145,errgroup.go:57: error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:399
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1820
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1784
		    main.runTPCC.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:157
		    main.(*monitor).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2145
		    github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1656753-1576823732-03-n4cpu16:4 -- ./workload run tpcc --warehouses=1350 --histograms=perf/stats.json  --ramp=5m0s --duration=144h0m0s {pgurl:1-3} returned:
		    stderr:
		    
		    stdout:
		              23.8   1040.2   9126.8  21474.8  21474.8 stockLevel
		    73918.0s        0           13.0           23.8   4160.7  11274.3  16106.1  16106.1 delivery
		    73918.0s        0          190.6          237.5   4160.7  24696.1  57982.1  94489.3 newOrder
		    73918.0s        0           19.0           23.8    209.7   2281.7   9126.8   9126.8 orderStatus
		    73918.0s        0          127.7          237.5   6174.0  33286.0  57982.1  81604.4 payment
		    73918.0s        0           26.9           23.8    637.5   5100.3  11811.2  11811.2 stockLevel
		    73919.0s        0           28.1           23.8   3221.2   9663.7  12348.0  12348.0 delivery
		    73919.0s        0          243.5          237.5   4160.7  22548.6  45097.2 103079.2 newOrder
		    73919.0s        0           32.1           23.8    201.3   4831.8   8321.5   8321.5 orderStatus
		    73919.0s        0          323.6          237.5   4563.4  23622.3  38654.7  66572.0 payment
		    73919.0s        0           24.0           23.8   1006.6  10200.5  13421.8  13421.8 stockLevel
		    Error:  exit status 255:
		  - exit status 1
```

<details><summary>Repro</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656753&tab=artifacts#/weekly/tpcc-max)
```
make stressrace TESTS=weekly/tpcc-max PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1664300&tab=buildLog) on [master@d54e001248f233734ff926daef5470487c5616b0](https://github.com/cockroachdb/cockroach/commits/d54e001248f233734ff926daef5470487c5616b0):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191227-1664300/weekly/tpcc-max/run_1
	cluster.go:1786,tpcc.go:157,cluster.go:2145,errgroup.go:57: error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:399
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1820
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1784
		    main.runTPCC.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:157
		    main.(*monitor).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2145
		    github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1664300-1577428581-03-n4cpu16:4 -- ./workload run tpcc --warehouses=1350 --histograms=perf/stats.json  --ramp=5m0s --duration=144h0m0s {pgurl:1-3} returned:
		    stderr:
		    
		    stdout:
		           8.0           21.5  10737.4  16106.1  16106.1  16106.1 delivery
		    73460.0s        0          118.7          215.3  13958.6  90194.3 103079.2 103079.2 newOrder
		    73460.0s        0            9.0           21.5   4026.5 103079.2 103079.2 103079.2 orderStatus
		    73460.0s        0           94.8          215.3   7784.6  94489.3 103079.2 103079.2 payment
		    73460.0s        0           11.0           21.5   7784.6  94489.3 103079.2 103079.2 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    73461.0s        0           11.0           21.5  18253.6  90194.3 103079.2 103079.2 delivery
		    73461.0s        0          132.3          215.3  13421.8  90194.3 103079.2 103079.2 newOrder
		    73461.0s        0            9.0           21.5   3355.4  94489.3  94489.3  94489.3 orderStatus
		    73461.0s        0          121.3          215.3   9663.7  94489.3 103079.2 103079.2 payment
		    73461.0s        0           10.0           21.5   3489.7 103079.2 103079.2 103079.2 stockLevel
		    Error:  exit status 255:
		  - exit status 1
```

<details><summary>Repro</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1664300&tab=artifacts#/weekly/tpcc-max)
```
make stressrace TESTS=weekly/tpcc-max PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671793&tab=buildLog) on [master@8facb469d66426a3f372a1aed4ea6f34e287b501](https://github.com/cockroachdb/cockroach/commits/8facb469d66426a3f372a1aed4ea6f34e287b501):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200103-1671793/weekly/tpcc-max/run_1
	cluster.go:1809,tpcc.go:157,cluster.go:2168,errgroup.go:57: error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:399
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1843
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1807
		    main.runTPCC.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:157
		    main.(*monitor).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2168
		    github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1671793-1578033389-03-n4cpu16:4 -- ./workload run tpcc --warehouses=1350 --histograms=perf/stats.json  --ramp=5m0s --duration=144h0m0s {pgurl:1-3} returned:
		    stderr:
		    
		    stdout:
		           9.0           21.8   9126.8  47244.6  47244.6  47244.6 delivery
		    71580.0s        0          109.9          217.5  10200.5 103079.2 103079.2 103079.2 newOrder
		    71580.0s        0           13.0           21.8    469.8  36507.2 103079.2 103079.2 orderStatus
		    71580.0s        0          140.9          217.6   8321.5 103079.2 103079.2 103079.2 payment
		    71580.0s        0           18.0           21.8   2415.9  24696.1  26843.5  26843.5 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    71581.0s        0           14.0           21.8  21474.8  73014.4 103079.2 103079.2 delivery
		    71581.0s        0          135.4          217.5   9663.7  94489.3 103079.2 103079.2 newOrder
		    71581.0s        0           16.0           21.8    436.2  21474.8  42949.7  42949.7 orderStatus
		    71581.0s        0          125.3          217.6   5637.1 103079.2 103079.2 103079.2 payment
		    71581.0s        0            8.0           21.8   8053.1 103079.2 103079.2 103079.2 stockLevel
		    Error:  exit status 255:
		  - exit status 1
```

<details><summary>Repro</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671793&tab=artifacts#/weekly/tpcc-max)
```
make stressrace TESTS=weekly/tpcc-max PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1683224&tab=buildLog) on [master@413fd1e46c2936029458db495efa242acb1b7d52](https://github.com/cockroachdb/cockroach/commits/413fd1e46c2936029458db495efa242acb1b7d52):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200110-1683224/weekly/tpcc-max/run_1
	cluster.go:1809,tpcc.go:157,cluster.go:2168,errgroup.go:57: error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:399
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1843
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1807
		    main.runTPCC.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:157
		    main.(*monitor).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2168
		    github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1683224-1578639885-02-n4cpu16:4 -- ./workload run tpcc --warehouses=1350 --histograms=perf/stats.json  --ramp=5m0s --duration=144h0m0s {pgurl:1-3} returned:
		    stderr:
		    
		    stdout:
		     26.2   6442.5  11811.2  12348.0  12348.0 delivery
		     8469.0s        0          510.7          261.9   6174.0   9663.7  10737.4  19327.4 newOrder
		     8469.0s        0           14.0           26.2   1946.2   2952.8   3758.1   3758.1 orderStatus
		     8469.0s        0            1.0          262.3   5100.3   5100.3   5100.3   5100.3 payment
		     8469.0s        0           14.0           26.2   1744.8   5905.6   8589.9   8589.9 stockLevel
		     8470.0s        0            8.0           26.2   8053.1  11274.3  11274.3  11274.3 delivery
		     8470.0s        0           30.1          261.9   6710.9   8321.5   9126.8   9126.8 newOrder
		     8470.0s        0           11.0           26.2   3489.7   4563.4   4563.4   4563.4 orderStatus
		     8470.0s        0            0.0          262.3      0.0      0.0      0.0      0.0 payment
		     8470.0s        0            8.0           26.2   3355.4   4160.7   4160.7   4160.7 stockLevel
		    Error: error in newOrder: ERROR: result is ambiguous (intent missing and record aborted) (SQLSTATE 40003)
		    Error:  exit status 1:
		  - exit status 1
```

<details><summary>Repro</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1683224&tab=artifacts#/weekly/tpcc-max)
```
make stressrace TESTS=weekly/tpcc-max PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1720327&tab=buildLog) on [master@9e290a564deda51b1db8867e4d0493f3ec1d1d96](https://github.com/cockroachdb/cockroach/commits/9e290a564deda51b1db8867e4d0493f3ec1d1d96):

```
		    18446.0s        0            0.0          281.0      0.0      0.0      0.0      0.0 payment
		    18446.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 stockLevel
		    18447.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 delivery
		    18447.0s        0            1.0          280.8  15032.4  15032.4  15032.4  15032.4 newOrder
		    18447.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 orderStatus
		    18447.0s        0            0.0          281.0      0.0      0.0      0.0      0.0 payment
		    18447.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 stockLevel
		    18448.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 delivery
		    18448.0s        0            1.0          280.8  15032.4  15032.4  15032.4  15032.4 newOrder
		    18448.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 orderStatus
		    18448.0s        0            0.0          281.0      0.0      0.0      0.0      0.0 payment
		    18448.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    18449.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 delivery
		    18449.0s        0            0.0          280.7      0.0      0.0      0.0      0.0 newOrder
		    18449.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 orderStatus
		    18449.0s        0            0.0          280.9      0.0      0.0      0.0      0.0 payment
		    18449.0s        0            0.0           28.1      0.0      0.0      0.0      0.0 stockLevel:
		  - exit status 1

	cluster.go:2368,tpcc.go:168,tpcc.go:296,test_runner.go:734: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.registerTPCC.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:296
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:734
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1720327&tab=artifacts#/weekly/tpcc-max)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aweekly%2Ftpcc-max.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1734652&tab=buildLog) on [master@b67d7d14e22f227caa4a2bc3727e3baf815fb25e](https://github.com/cockroachdb/cockroach/commits/b67d7d14e22f227caa4a2bc3727e3baf815fb25e):

```
		    82071.0s        0           67.0          215.2  12884.9 103079.2 103079.2 103079.2 payment
		    82071.0s        0           13.0           21.5   6710.9 103079.2 103079.2 103079.2 stockLevel
		    82072.0s        0            7.0           21.5  53687.1 103079.2 103079.2 103079.2 delivery
		    82072.0s        0          105.9          215.1  16106.1 103079.2 103079.2 103079.2 newOrder
		    82072.0s        0            6.0           21.5    503.3 103079.2 103079.2 103079.2 orderStatus
		    82072.0s        0          108.9          215.2   9663.7 103079.2 103079.2 103079.2 payment
		    82072.0s        0            7.0           21.5  13958.6 103079.2 103079.2 103079.2 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    82073.0s        0           18.0           21.5  20401.1 103079.2 103079.2 103079.2 delivery
		    82073.0s        0          138.1          215.1  17179.9 103079.2 103079.2 103079.2 newOrder
		    82073.0s        0            6.0           21.5  15032.4  31138.5  31138.5  31138.5 orderStatus
		    82073.0s        0          160.1          215.2  10737.4 103079.2 103079.2 103079.2 payment
		    82073.0s        0            5.0           21.5  77309.4 103079.2 103079.2 103079.2 stockLevel
		    82074.0s        0           15.0           21.5  32212.3  94489.3 103079.2 103079.2 delivery
		    82074.0s        0          123.9          215.1  18253.6 103079.2 103079.2 103079.2 newOrder
		    82074.0s        0           16.0           21.5   4831.8  85899.3  90194.3  90194.3 orderStatus
		    82074.0s        0          116.9          215.2  19327.4 103079.2 103079.2 103079.2 payment
		    82074.0s        0            5.0           21.5  11274.3  22548.6  22548.6  22548.6 stockLevel:
		  - exit status 1

	cluster.go:2368,tpcc.go:168,tpcc.go:296,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.registerTPCC.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:296
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1734652&tab=artifacts#/weekly/tpcc-max)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aweekly%2Ftpcc-max.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1749354&tab=buildLog) on [master@e671a4ef97cbc6cf5d22f8f322fd45733d302094](https://github.com/cockroachdb/cockroach/commits/e671a4ef97cbc6cf5d22f8f322fd45733d302094):

```
		    344515.0s        0          235.0          212.6  13958.6  33286.0  53687.1  68719.5 payment
		    344515.0s        0           11.0           21.3  13421.8  22548.6  31138.5  31138.5 stockLevel
		    344516.0s        0           17.0           21.3  14495.5  28991.0  36507.2  36507.2 delivery
		    344516.0s        0          164.2          212.6  13421.8  40802.2  64424.5  90194.3 newOrder
		    344516.0s        0           12.0           21.3   2281.7  19327.4  19327.4  19327.4 orderStatus
		    344516.0s        0          121.1          212.6  15569.3  45097.2  66572.0  90194.3 payment
		    344516.0s        0           14.0           21.3   5100.3  19327.4  22548.6  22548.6 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    344517.0s        0            9.0           21.3   5368.7  20401.1  20401.1  20401.1 delivery
		    344517.0s        0          114.0          212.6  13958.6  31138.5  42949.7  94489.3 newOrder
		    344517.0s        0           12.0           21.3   4831.8  18253.6  31138.5  31138.5 orderStatus
		    344517.0s        0          213.9          212.6  12884.9  40802.2  66572.0  90194.3 payment
		    344517.0s        0           19.0           21.3  11274.3  19327.4  19327.4  19327.4 stockLevel
		    344518.0s        0           29.0           21.3  11274.3  32212.3  36507.2  36507.2 delivery
		    344518.0s        0          174.0          212.6  13421.8  33286.0  73014.4  77309.4 newOrder
		    344518.0s        0           18.0           21.3   4160.7  18253.6  18253.6  18253.6 orderStatus
		    344518.0s        0          239.0          212.6  15032.4  40802.2  57982.1  90194.3 payment
		    344518.0s        0           19.0           21.3   6710.9  19327.4  22548.6  22548.6 stockLevel:
		  - exit status 1

	cluster.go:2368,tpcc.go:168,tpcc.go:296,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.registerTPCC.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:296
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1749354&tab=artifacts#/weekly/tpcc-max)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aweekly%2Ftpcc-max.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1759628&tab=buildLog) on [master@b797cad6d130714748983bc53d4611ddc6151153](https://github.com/cockroachdb/cockroach/commits/b797cad6d130714748983bc53d4611ddc6151153):

```
		    47240.0s        0          156.6          242.3  21474.8  73014.4 103079.2 103079.2 payment
		    47240.0s        0           11.0           24.2  26843.5  38654.7  47244.6  47244.6 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    47241.0s        0           24.1           24.2  22548.6  51539.6  53687.1  53687.1 delivery
		    47241.0s        0          153.5          242.3  19327.4  53687.1 103079.2 103079.2 newOrder
		    47241.0s        0           19.1           24.2  13958.6  26843.5  36507.2  36507.2 orderStatus
		    47241.0s        0          146.4          242.3  17179.9  60129.5 103079.2 103079.2 payment
		    47241.0s        0           21.1           24.2  11811.2  47244.6  53687.1  53687.1 stockLevel
		    47242.0s        0           13.0           24.2  14495.5  51539.6  53687.1  53687.1 delivery
		    47242.0s        0          139.8          242.3  20401.1  55834.6 103079.2 103079.2 newOrder
		    47242.0s        0           11.0           24.2   9663.7  18253.6  23622.3  23622.3 orderStatus
		    47242.0s        0          129.8          242.3  19327.4  77309.4 103079.2 103079.2 payment
		    47242.0s        0            9.0           24.2  25769.8  40802.2  40802.2  40802.2 stockLevel
		    47243.0s        0           18.0           24.2  16106.1  47244.6  47244.6  47244.6 delivery
		    47243.0s        0          108.2          242.3  20401.1  51539.6 103079.2 103079.2 newOrder
		    47243.0s        0           11.0           24.2  11274.3  25769.8  26843.5  26843.5 orderStatus
		    47243.0s        0          106.2          242.3  18253.6  47244.6  55834.6 103079.2 payment
		    47243.0s        0           11.0           24.2  21474.8  68719.5  77309.4  77309.4 stockLevel:
		  - exit status 1

	cluster.go:2368,tpcc.go:168,tpcc.go:296,test_runner.go:741: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.registerTPCC.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:296
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:741
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1759628&tab=artifacts#/weekly/tpcc-max)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aweekly%2Ftpcc-max.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1806443&tab=buildLog) on [master@69dc87d68addedf2fabfb2b14c098cfb35b5f3d0](https://github.com/cockroachdb/cockroach/commits/69dc87d68addedf2fabfb2b14c098cfb35b5f3d0):

```
		    69007.0s        0          294.7          245.3   1140.9   6442.5  26843.5  26843.5 payment
		    69007.0s        0           22.1           24.5   1275.1   9126.8  27917.3  27917.3 stockLevel
		    69008.0s        0           32.0           24.5   1811.9   8053.1  31138.5  31138.5 delivery
		    69008.0s        0          298.0          245.2   1409.3   3892.3  27917.3  27917.3 newOrder
		    69008.0s        0           27.0           24.5    134.2    402.7    436.2    436.2 orderStatus
		    69008.0s        0          281.0          245.3    906.0   5100.3   8589.9 103079.2 payment
		    69008.0s        0           26.0           24.5   1342.2   5637.1  10737.4  10737.4 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    69009.0s        0           27.9           24.5   1879.0   5100.3   8321.5   8321.5 delivery
		    69009.0s        0          253.4          245.2   1342.2   3758.1  11274.3 103079.2 newOrder
		    69009.0s        0           22.0           24.5    142.6    285.2    335.5    335.5 orderStatus
		    69009.0s        0          266.4          245.3    939.5   4026.5  11811.2  27917.3 payment
		    69009.0s        0           19.0           24.5    738.2   4563.4   6979.3   6979.3 stockLevel
		    69010.0s        0           33.0           24.5   1811.9   2550.1   7247.8   7247.8 delivery
		    69010.0s        0          265.0          245.2   1275.1   3892.3  28991.0 103079.2 newOrder
		    69010.0s        0           24.0           24.5    109.1    335.5    335.5    335.5 orderStatus
		    69010.0s        0          227.0          245.3    906.0   4295.0  28991.0  28991.0 payment
		    69010.0s        0           22.0           24.5    872.4   3087.0   8053.1   8053.1 stockLevel:
		  - exit status 1

	cluster.go:2368,tpcc.go:168,tpcc.go:296,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.registerTPCC.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:296
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1806443&tab=artifacts#/weekly/tpcc-max)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aweekly%2Ftpcc-max.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1819500&tab=buildLog) on [master@bcffc157e3af316c2bfcf4850e67205010425211](https://github.com/cockroachdb/cockroach/commits/bcffc157e3af316c2bfcf4850e67205010425211):

```
		    74792.0s        0            1.0           76.8     13.6     13.6     13.6     13.6 payment
		    74792.0s        0            0.0            7.7      0.0      0.0      0.0      0.0 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    74793.0s        0            0.0            7.7      0.0      0.0      0.0      0.0 delivery
		    74793.0s        0            1.0           76.8     32.5     32.5     32.5     32.5 newOrder
		    74793.0s        0            0.0            7.7      0.0      0.0      0.0      0.0 orderStatus
		    74793.0s        0            2.0           76.8     13.6     14.7     14.7     14.7 payment
		    74793.0s        0            0.0            7.7      0.0      0.0      0.0      0.0 stockLevel
		    74794.0s        0            0.0            7.7      0.0      0.0      0.0      0.0 delivery
		    74794.0s        0            0.0           76.8      0.0      0.0      0.0      0.0 newOrder
		    74794.0s        0            0.0            7.7      0.0      0.0      0.0      0.0 orderStatus
		    74794.0s        0            2.0           76.8     18.9     19.9     19.9     19.9 payment
		    74794.0s        0            2.0            7.7     41.9     60.8     60.8     60.8 stockLevel
		    74795.0s        0            0.0            7.7      0.0      0.0      0.0      0.0 delivery
		    74795.0s        0            4.0           76.8     29.4     30.4     30.4     30.4 newOrder
		    74795.0s        0            1.0            7.7      8.9      8.9      8.9      8.9 orderStatus
		    74795.0s        0            3.0           76.8     17.8     30.4     30.4     30.4 payment
		    74795.0s        0            0.0            7.7      0.0      0.0      0.0      0.0 stockLevel:
		  - exit status 1

	cluster.go:2368,tpcc.go:168,tpcc.go:296,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.registerTPCC.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:296
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1819500&tab=artifacts#/weekly/tpcc-max)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aweekly%2Ftpcc-max.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).weekly/tpcc-max failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1832575&tab=buildLog) on [master@c0855e9fcf979cca1094d7edeeacb9882ddc9f96](https://github.com/cockroachdb/cockroach/commits/c0855e9fcf979cca1094d7edeeacb9882ddc9f96):

```
		    68946.0s        0            1.0          199.3     12.6     12.6     12.6     12.6 payment
		    68946.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 stockLevel
		    68947.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 delivery
		    68947.0s        0            0.0          199.2      0.0      0.0      0.0      0.0 newOrder
		    68947.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 orderStatus
		    68947.0s        0            0.0          199.3      0.0      0.0      0.0      0.0 payment
		    68947.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 stockLevel
		    68948.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 delivery
		    68948.0s        0            0.0          199.2      0.0      0.0      0.0      0.0 newOrder
		    68948.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 orderStatus
		    68948.0s        0            0.0          199.3      0.0      0.0      0.0      0.0 payment
		    68948.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 stockLevel
		    _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    68949.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 delivery
		    68949.0s        0            0.0          199.2      0.0      0.0      0.0      0.0 newOrder
		    68949.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 orderStatus
		    68949.0s        0            0.0          199.2      0.0      0.0      0.0      0.0 payment
		    68949.0s        0            0.0           19.9      0.0      0.0      0.0      0.0 stockLevel:
		  - exit status 30

	cluster.go:2368,tpcc.go:168,tpcc.go:296,test_runner.go:753: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2356
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2364
		    main.runTPCC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:168
		    main.registerTPCC.func4
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:296
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2412
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.init
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2309
		    runtime.doInit
		    	/usr/local/go/src/runtime/proc.go:5222
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:190
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - Goexit() was called

```

<details><summary>More</summary><p>


Artifacts: [/weekly/tpcc-max](https://teamcity.cockroachdb.com/viewLog.html?buildId=1832575&tab=artifacts#/weekly/tpcc-max)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aweekly%2Ftpcc-max.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/tpcc.go
+++ b/pkg/cmd/roachtest/tpcc.go
@@ -285,14 +285,14 @@ func registerTPCC(r *testRegistry) {
 		},
 	})
 	r.Add(testSpec{
-		Name:       "weekly/tpcc-max",
+		Name:       "weekly/tpcc/headroom",
 		Owner:      OwnerKV,
 		MinVersion: maybeMinVersionForFixturesImport(cloud),
 		Tags:       []string{`weekly`},
 		Cluster:    makeClusterSpec(4, cpu(16)),
 		Timeout:    time.Duration(6*24)*time.Hour + time.Duration(10)*time.Minute,
 		Run: func(ctx context.Context, t *test, c *cluster) {
-			warehouses := 1350
+			warehouses := 1000
 			runTPCC(ctx, t, c, tpccOptions{
 				Warehouses: warehouses,
 				Duration:   6 * 24 * time.Hour,

